lta/picker.py

```diff
@@ -53,10 +56,13 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
                 raise ValueError(f"Missing expected configuration parameter: '{name}'")
         # assimilate provided configuration
         self.file_catalog_rest_url = config["FILE_CATALOG_REST_URL"]
-        self.heartbeat_sleep_duration_seconds = int(config["HEARTBEAT_SLEEP_DURATION_SECONDS"])
+        self.heartbeat_patch_retries = int(config["HEARTBEAT_PATCH_RETRIES"])
+        self.heartbeat_patch_timeout_seconds = float(config["HEARTBEAT_PATCH_TIMEOUT_SECONDS"])
+        self.heartbeat_sleep_duration_seconds = float(config["HEARTBEAT_SLEEP_DURATION_SECONDS"])
+        self.lta_rest_token = config["LTA_REST_TOKEN"]
         self.lta_rest_url = config["LTA_REST_URL"]
         self.picker_name = config["PICKER_NAME"]
-        self.work_sleep_duration_seconds = int(config["WORK_SLEEP_DURATION_SECONDS"])
+        self.work_sleep_duration_seconds = float(config["WORK_SLEEP_DURATION_SECONDS"])
```

Comment:
Why float here?  Is this so testing can specify sub-1 second times?

Reply:
This was motivated first by this idea:

- My configuration had added a true discrete quantity (retries); or, I don't think 2.73 retries has a sensible interpretation. So this one can and should be an `int()`

This made me reconsider the other configuration types:

- Seconds is a continuous quantity where someone could (in theory) provide something like 20.5 seconds. Why they would do it, who knows, but the code would be more tolerant if it also accept float values. So this one can and should be a `float()`.